### DISCUSSION TITLE
请问某个域名指定IPV4地址和禁用IPV6是一条设置可以完成的吗？

### DISCUSSION THREAD

**Discussion by imtv** [pymumu/smartdns#1430] [Q&A] [answered]
address /abc.com/1.1.1.1
address /abc.com/#6

请问这两条可以合并吗？

根据文档的格式尝试了一下
都失败了，求助求助

**Comment by PikuZheng** [comment, marked as answer]
`domain-rules /abc.com/ -a 1.1.1.1 -a #6`

**Reply by imtv** [reply]
谢谢
